deploy/sre-prometheus/management-cluster/100-dynatrace-workloads-monitoring.PrometheusRule.yaml

```diff
@@ -8,151 +8,148 @@ metadata:
-      annotations:
-        summary: "Dynatrace OTEL Collector is Degraded"
-        description: "{{$value}} replicas are not running out of total required replicas for the OpenTelemetry Dynatrace Collector"
-        SOP : https://github.com/openshift/ops-sop/blob/master/dynatrace/alerts/DynatraceOpenTelemetryCollectorDegradedSRE.md
+        - alert: DynatraceWebhookDegradedSRE
+          expr: (sum(kube_deployment_spec_replicas{deployment="dynatrace-webhook", namespace="dynatrace"}) without (deployment, instance, pod)) - (sum(kube_deployment_status_replicas_available{deployment="dynatrace-webhook", namespace="dynatrace"}) without (deployment, instance, pod)) > 0
+          for: 15m
+          labels:
+            namespace: dynatrace
+            severity: warning
+          annotations:
+            summary: "Dynatrace Webhook is Degraded"
+            description: "{{$value}} replicas are not running out of total required replicas for the Dynatrace Webhook"
+            SOP: https://github.com/openshift/ops-sop/blob/master/dynatrace/alerts/DynatraceWebhookDegradedSRE.md
 
-    - alert: DynatracePodRestartsStuckSRE
-      expr: kube_pod_container_status_restarts_total{namespace="dynatrace"} > 0 and increase(kube_pod_container_status_restarts_total{namespace="dynatrace"}[15m]) > 0
-      for: 15m
-      labels:
-        namespace: dynatrace
-        severity: warning
-      annotations:
-        summary: "Pod Restarts Stuck Alert"
-        description: "At least one pod in the 'dynatrace' namespace has been stuck in restarts for the last 15 minutes."
-        SOP : https://github.com/openshift/ops-sop/blob/master/dynatrace/alerts/DynatracePodRestartsStuckSRE.md
-    
-    - alert: DynatraceDynakubeComponenetsDegradedSRE
-      expr: count(kube_daemonset_status_desired_number_scheduled{namespace="dynatrace", daemonset=~"hs-mc.+"})
-      for: 60m
-      labels:
-        namespace: dynatrace
-        severity: critical
-        daemonset: hcp-workload-oneagent
-        state: degraded
-      annotations:
-        summary: "HCP workload oneagent daemonset missing"
-        description: "HCP workload oneagent daemonset in the 'dynatrace' namespace is missing for 60 minutes."
-        SOP : https://github.com/openshift/ops-sop/blob/master/dynatrace/docs/troubleshooting/dynakube.md
+        - alert: DynatraceOpenTelemetryCollectorDegradedSRE
+          expr: kube_deployment_status_replicas_unavailable{namespace="dynatrace", deployment="custom-opentelemetry-dynatrace-collector"}  > 0
+          for: 5m
+          labels:
+            namespace: dynatrace
+            severity: critical
+          annotations:
+            summary: "Dynatrace OTEL Collector is Degraded"
+            description: "{{$value}} replicas are not running out of total required replicas for the OpenTelemetry Dynatrace Collector"
+            SOP: https://github.com/openshift/ops-sop/blob/master/dynatrace/alerts/DynatraceOpenTelemetryCollectorDegradedSRE.md
```

Comment:
```suggestion
            SOP: https://github.com/openshift/ops-sop/blob/master/dynatrace/alerts/DynatraceOpenTelemetryCollectorDownSRE.md
```

Reply:
We can rename the file as separate PR in ops-sop too. Mainly if there's a single instance of OTEL and it's not available then we can raise it as a "Down" state alert which is more severe. Once we leverage OTEL metrics better to detect degradation of the service then we can have a "Degraded" alert as well.